atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -46,6 +46,25 @@ def __init__(self, tasker, workflow, hub, target=None, koji_ssl_certs_dir=None):
             }
         self.xmlrpc = create_koji_session(hub, koji_auth_info)
 
+    def get_patched_release(self, original_release, increment=False):
+        if original_release.count('.') > 1:
+            release, suffix, remainder = original_release.split('.', 2)
+        else:
+            remainder = ''
+            if original_release.count('.') == 1:
+                release, suffix = original_release.split('.', 1)
+            else:
+                release = original_release
+                suffix = ''
+
+        result = [str(int(release) + 1)] if increment else [release]
```

Comment:
It looks better, but I'm not sure if:
```if increment or is_dotrel:``` is a good idea - this would cause increment in first `self.get_patched_release(self.xmlrpc.getNextRelease(build_info))` call. That would fail the case from https://github.com/projectatomic/atomic-reactor/pull/769#discussion_r132246260, but I think its safe to assume that `20.fc25` exists if `20.1.fc25` is returned by `getNextRelease`, isn't it?